documentation/book/ref-kafka-bridge-tls.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-tls.adoc
+
+[id='ref-kafka-bridge-tls-{context}']
+= TLS support in Kafka Bridge
```

Comment:
You should make it clear here that the TLS here is only about the Kafka connection. Not about the HTTP bridge. Ideally in the title, but at least in the following paragraph.